### ISSUE TITLE
Video widget not working with Lightbox

### ISSUE THREAD

**Issue by pensierinmusica** [elementor/elementor#15080]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest development version of Elementor.

## Description

Hi, the Elementor "video" widget is not working, when you add an "Image Overlay" and turn on the "Lightbox" switch. When you click on the video in the rendered page, it doesn't play and you get this error in the browser's console:

```
frontend.min.js?ver=3.3.0-dev7:2 Uncaught (in promise) ReferenceError: elementor is not defined
    at LightboxManager._callee$ (frontend.min.js?ver=3.3.0-dev7:2)
    at tryCatch (frontend.min.js?ver=3.3.0-dev7:2)
    at Generator.invoke [as _invoke] (frontend.min.js?ver=3.3.0-dev7:2)
    at Generator.next (frontend.min.js?ver=3.3.0-dev7:2)
    at asyncGeneratorStep (frontend.min.js?ver=3.3.0-dev7:2)
    at _next (frontend.min.js?ver=3.3.0-dev7:2)
    at frontend.min.js?ver=3.3.0-dev7:2
    at new Promise (<anonymous>)
    at new F (frontend-modules.min.js?ver=3.3.0-dev7:2)
    at LightboxManager.<anonymous> (frontend.min.js?ver=3.3.0-dev7:2)
```

The issue comes from the "Lightbox" switch, otherwise everything else is working fine. I've disabled all caching and all plugins (apart from Elementor) but the issue still persists.

Can you please fix it asap? This is quite important!

## Steps to reproduce

Add a video widget, set an image overlay for it and turn on the "Lightbox" option. Then go to the rendered page and try to play the video.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [x] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.32-35
	PHP Version: 7.3.28
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.2
	Site URL: https://codeworks.me
	Home URL: https://codeworks.me
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 768M
	Permalink Structure: /blog/%postname%/
	Language: en-US
	Timezone: Europe/Madrid
	Debug Mode: Inactive

== Theme ==
	Name: Astra
	Version: 3.4.6
	Author: Brainstorm Force
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36

== Active Plugins ==
	301 Redirects
		Version: 2.70
		Author: WebFactory Ltd

	Astra Pro
		Version: 3.4.2
		Author: Brainstorm Force

	Custom Adobe Fonts (Typekit)
		Version: 1.0.18
		Author: Brainstorm Force

	Disable auto-update Email Notifications
		Version: 1.3.0
		Author: Blogies Tools

	DynamicConditions
		Version: 1.5.1
		Author: RTO GmbH

	Elementor
		Version: 3.3.0-dev7
		Author: Elementor.com

	Elementor Pro
		Version: 3.4.0-dev6
		Author: Elementor.com

	Image optimization service by Optimole
		Version: 3.1.0
		Author: Optimole

	Join.chat
		Version: 4.1.14
		Author: Creame

	Make Column Clickable Elementor
		Version: 1.3.1
		Author: Fernando Acosta

	Optimize Database after Deleting Revisions
		Version: 5.0.4
		Author: CAGE Web Design | Rolf van Gelder, Eindhoven, The Netherlands

	Responsive Charts
		Version: 1.7.9
		Author: Weblator

	SG Optimizer
		Version: 5.7.20
		Author: SiteGround

	Shortcoder
		Version: 5.4
		Author: Aakash Chakravarthy

	Simple Custom CSS and JS
		Version: 3.36
		Author: SilkyPress.com

	Ultimate Addons for Elementor
		Version: 1.31.0
		Author: Brainstorm Force

	Wordfence Security
		Version: 7.5.3
		Author: Wordfence

	WordPress Importer
		Version: 0.7
		Author: wordpressdotorg

	Yoast Duplicate Post
		Version: 4.1.2
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO
		Version: 16.3
		Author: Team Yoast


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com


== Elementor - Compatibility Tag ==
	
	Elementor Pro: incompatible
	Make Column Clickable Elementor: header_not_exists
	Ultimate Addons for Elementor: incompatible

== Elementor Pro - Compatibility Tag ==
	
	Ultimate Addons for Elementor: incompatible

== Elements Usage ==
	
	page : 1
		global : 2
		text-editor : 6
		uael-table : 6
		video : 6
	popup : 5
		button : 1
		heading : 1
	section : 51
		button : 30
		divider : 4
		global : 62
		heading : 11
		icon : 41
		icon-list : 11
		image : 40
		nav-menu : 11
		reviews : 2
		shortcode : 2
		social-icons : 1
		spacer : 16
		star-rating : 10
		text-editor : 199
		toggle : 2
		uael-image-gallery : 8
		uael-table : 3
		video : 4
	widget : 0
		accordion : 4
		alert : 1
		image : 1
		reviews : 2
		spacer : 6
		star-rating : 1
		text-editor : 4
		toggle : 2
		uael-image-gallery : 1
		uael-table : 6
		uael-timeline : 9
	wp-post : 10
		accordion : 4
		button : 87
		global : 509
		heading : 20
		html : 2
		icon : 44
		icon-list : 55
		image : 44
		image-carousel : 6
		menu-anchor : 170
		nav-menu : 20
		shortcode : 138
		spacer : 34
		star-rating : 19
		template : 17
		text-editor : 536
		toggle : 4
		uael-google-map : 1
		uael-image-gallery : 1
		uael-table : 23
		video : 31
	wp-page : 41
		button : 1
		divider : 2
		global : 4
		heading : 1
		posts : 4
		spacer : 1
		text-editor : 4


== Log ==
	
Log: showing 20 of 432021-05-25 09:32:11 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (9 of 24, 38%) 
2021-05-25 09:32:12 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (10 of 24, 42%) 
2021-05-25 09:32:13 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (11 of 24, 46%) 
2021-05-25 09:32:14 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (12 of 24, 50%) 
2021-05-25 09:32:15 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (13 of 24, 55%) 
2021-05-25 09:32:16 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (14 of 24, 59%) 
2021-05-25 09:32:17 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (15 of 24, 63%) 
2021-05-25 09:32:18 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (16 of 24, 67%) 
2021-05-25 09:32:19 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (17 of 24, 71%) 
2021-05-25 09:32:20 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (18 of 24, 75%) 
2021-05-25 09:32:21 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (19 of 24, 80%) 
2021-05-25 09:32:23 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (20 of 24, 84%) 
2021-05-25 09:32:24 [info] elementor-pro::elementor_pro_updater Started 
2021-05-25 09:32:24 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (21 of 24, 88%) 
2021-05-25 09:32:26 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (22 of 24, 92%) 
2021-05-25 09:32:27 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (23 of 24, 96%) 
2021-05-25 09:32:28 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start (24 of 24, 100%) 
2021-05-25 09:32:30 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-05-25 09:32:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.0-dev4',
  'to' => '3.4.0-dev6',
)]
2021-05-25 09:32:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.0-dev4',
  'to' => '3.4.0-dev6',
)]
```
</details>

**Comment by pensierinmusica** [comment]
@kobizz @KingYes @matipojo @iNewLegend @arielk @udidol @joshmarom @rotemee @Nevoss could any of you look into this asap?

I know you're super busy, but this is one of the top elements on our home page and it's completely breaking the UX.

Thanks!!

**Comment by pensierinmusica** [comment]
It seems that the `elementor` variable is not in the scope of that function. And looking at the global scope, I can only access the following:

```
elementorAppConfig
elementorAppLoader
elementorAppProConfig
elementorCommon
elementorCommonConfig
elementorFrontend
elementorFrontendConfig
elementorModules
elementorProFrontend
ElementorProFrontendConfig
webpackChunkelementor
webpackChunkelementor_pro
```

**Comment by pensierinmusica** [comment]
I opened a new issue, since the problem since to be wider than I initally thought.

https://github.com/elementor/elementor/issues/15085

**Comment by pensierinmusica** [comment]
For anyone who's interested, the issue seems to be present with Elementor 3.3.0-dev7.

I've reverted to the stable version and it works fine.

**Comment by shilo-ey** [comment]
Thanks, closing this issue because you got answered in the new one.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.3.0

Feel free to update

Thanks!
